operations/service.go

```diff
@@ -49,6 +49,8 @@ func parseDB(c *cli.Context) *evergreen.DBSettings {
 	if url == evergreen.DefaultDatabaseUrl && envUrl != "" {
 		url = envUrl
 	}
+	envUser := os.Getenv(evergreen.MongodbUser)
```

Comment:
asking inline:
What is the plan for getting the creds into the environment?

Reply:
Several options:
1) Just set it manually, don't write down anywhere (probably overkill)
2) put it in the evergreen-prod.env file in a private repo, plaintext, and deploy as usual (easiest, but least secure)
3) check a gpg-encrypted password file into a private repo, and unlock this before we deploy with a password we all memorize (compromise solution)
4) use some kind of secret manager like Vault instead (probably overkill)